tutorial/example_repo/workflows.py

```diff
@@ -28,6 +28,18 @@
 # workflows.
 FINAL_JOB = CommandJobTemplate('final', 'echo success')
 
+# Pinball allows an upstream job to pass attributes to downstream dependants.
+# Pinball executor has the ability to interpret specially formatted job output.
+# E.g., printing PINBALL:EVENT_ATTR:akey=avalue to the job output will embed a
+# akey-avalue pair in the job output event. Output events traverse along job
+# output edges and they are accessible by downstream jobs. Event attributes are
+# used to customize the job command line. E.g., command “echo %(akey)s” will be
+# translated to “echo avalue” by the job executor. For more details, please try
+# workflow "pass_params_between_parent_and_child_jobs" listed in WORKFLOWS
+PARENT_JOB = CommandJobTemplate('parent', 'echo PINBALL:EVENT_ATTR:akey=avalue')
```

Comment:
Rename this to something context dependent. Also, for consistency I would prefer to inline the job definition.